### DISCUSSION TITLE
Simple client template to get started?

### DISCUSSION THREAD

**Discussion by SuperSonicHub1** [Instawork/hyperview#870] [Q&A]
I'm interested in giving Hyperview a shot, but would rather not fork the whole of this repository to do so. Has the React Native client been turned into a library so it can be easily imported in an Expo project? If so, I'd be happy to make a template so people can more easily jump into Hyperview development.

**Comment by flochtililoch** [comment]
Hi @SuperSonicHub1, thanks for your interest in contributing to Hyperview!
We actually have [this open issue](https://github.com/Instawork/hyperview/issues/451) to facilitate creation of Hyperview apps through Expo templates. @Tobi-De had started this initiative earlier this year, though [his PR](https://github.com/Instawork/hyperview/pull/815) is still in draft, so I think there's still some work left, and we would love for you to revive this project!
Let us know how we can assist you, or if you have any question!
